railties/lib/rails/generators/rails/app/templates/Dockerfile.tt

```diff
@@ -20,14 +20,32 @@ RUN apt-get update -qq && \
     apt-get install -y <%= dockerfile_build_packages.join(" ") %>
 
 <% if using_node? -%>
-# Install JavaScript dependencies
-ARG NODE_VERSION=<%= dockerfile_node_version %>
-ARG YARN_VERSION=<%= dockerfile_yarn_version %>
-ENV VOLTA_HOME="/usr/local"
-RUN curl https://get.volta.sh | bash && \
-    volta install node@$NODE_VERSION yarn@$YARN_VERSION
+# Install JavaScript dependencies and libvips for Active Storage
+ ARG NODE_MAJOR_VERSION=19
+ RUN curl -sL https://deb.nodesource.com/setup_$NODE_MAJOR_VERSION.x | bash -
+ RUN apt-get update -qq && \
+     apt-get install -y build-essential libvips nodejs && \
+     apt-get clean && \
+     rm -rf /var/lib/apt/lists/* /usr/share/doc /usr/share/man && \
+     npm install -g yarn
+
+<% elsif !skip_active_storage? -%>
```

Comment:
The previous PR changed this to:

```erb
<% end -%>
<% if !skip_active_storage? -%>
```

Which there might be a good reason for, but I'm not sure.

Does Active Storage depend on node?

Reply:
Active Storage does not depend on node, but you don't need to install libvips twice, and you don't need to run apt-get install twice.

dockerfile-rails has switched to a strategy of determining which packages to install in the generator, all the template has to do is to iterate over the results.